### DISCUSSION TITLE
Upower tooltip need help

### DISCUSSION THREAD

**Discussion by LukashonakV** [Alexays/Waybar#3129] [Q&A] [answered]
Can someone provide screenshots or small video how does Upower module tooltip look like ? 
Thanks

**Comment by ordy** [comment, marked as answer]
Had to edit the code to have at least my BAT0 to show, here is what I have:

[upower.webm](https://github.com/Alexays/Waybar/assets/22659443/f0ef9854-d323-4dab-8951-19d04b1e2d2f)


~~(on light background)~~ ignore this, forgot to set tray menu css
![sc_709](https://github.com/Alexays/Waybar/assets/22659443/31f31f38-a50a-402d-92b8-c3f58e40b8f0)

(built from current master branch)

**Comment by LukashonakV** [comment]
Thank you, @ordy
